src/backend/model_deployments/azure.py

```diff
@@ -44,8 +44,16 @@ def __init__(self, **kwargs: Any):
             base_url=self.chat_endpoint_url, api_key=self.api_key
         )
 
-    @property
-    def rerank_enabled(self) -> bool:
+    @classmethod
+    def name(cls) -> str:
+        return "Azure"
+
+    @classmethod
+    def env_vars(cls) -> List[str]:
+        return AZURE_ENV_VARS
```

Comment:
Does this need to be a constant anymore? Just define the array right here

Reply:
Probably not